tests/test_utils/test_transformation.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+import pytest
+
+from lisdf.utils.transformations import euler_matrix
+from lisdf.utils.transformations_more import lookat_rpy
+
+
+@pytest.mark.parametrize(
+    "camera_pos, target_pos",
+    [
+        [[3, 8, 3], [0, 8, 1]],
+        [[3, 8, 3], [0, 9, 1]],
+        [[3, 8, 3], [0, 7, 1]],
+        [[0, 8, 1], [3, 8, 3]],
+        [[0, 9, 1], [3, 8, 3]],
+        [[0, 7, 1], [3, 8, 3]],
+    ],
+)
+def test_lookat_rpy(camera_pos, target_pos):
+    camera_pos = np.array(camera_pos, dtype="float32")
+    target_pos = np.array(target_pos, dtype="float32")
+    delta = target_pos - camera_pos
+    delta = delta / np.linalg.norm(delta)
+
+    rpy = lookat_rpy(camera_pos, target_pos)
```

Comment:
Can you add some comments here how the transformation is working for the asserts? Like what the `z` matrix and `zz` matrix is? My linear algebra is not the best... :)